types/module/interfaces/consensus_params.go

```diff
@@ -0,0 +1,12 @@
+package interfaces
```

Comment:
I think this is OK @tac0turtle?

Reply:
it's to avoid directly import [cometbft](https://github.com/cosmos/cosmos-sdk/pull/16583#discussion_r1243062336) type, sth like allow B get A without import X:
A import X
B import A